addons/im_livechat/demo/im_livechat_channel/im_livechat_session_1.xml

```diff
@@ -3,97 +3,112 @@
     <data>
         <record id="livechat_channel_session_1" model="discuss.channel">
             <field name="channel_type">livechat</field>
-            <field name="livechat_channel_id" ref="im_livechat_channel_data"/>
-            <field name="livechat_operator_id" ref="base.partner_admin"/>
+            <field name="livechat_channel_id" ref="im_livechat_channel_data" />
```

Comment:
All the spaces added were not needed and not preferred. Should be reverted so that actual changes can be easily reviewed.